assets/css/main.css

```diff
@@ -436,14 +436,21 @@ a:active {
     font-size: 1.3em;
 }
 .shelter-popup .shelter-details .content svg {
-    height: 1.4rem;
+    vertical-align: middle;
+    height: 1.2rem;
+}
+.shelter-details a {
+    text-decoration: underline;
 }
 .mapboxgl-popup-content svg {
     height: 1em;
 }
 .shelter-popup .mapboxgl-popup-close-button {
     font-size: 2.5em;
     line-height: .7em;
+}
+#icon-directions {
```

Comment:
Can I get you to do a `gulp build` then push the updated `main.css` and `main.css.map`?